extensions/resteasy-reactive/quarkus-resteasy-reactive-kotlin/deployment/src/main/java/io/quarkus/resteasy/reactive/kotlin/deployment/KotlinCoroutineIntegrationProcessor.java

```diff
@@ -185,4 +191,10 @@ public boolean isMethodSignatureAsync(MethodInfo info) {
             }
         });
     }
+
+    private static HandlerChainCustomizer flowCustomizer() {
+        return new FixedHandlersChainCustomizer(
+                List.of(new FlowToPublisherHandler(), new PublisherResponseHandler()),
```

Comment:
I can't find the FlowToPublisherHandler class anywhere.

Reply:
https://github.com/quarkusio/quarkus/blob/main/extensions/resteasy-reactive/quarkus-resteasy-reactive-kotlin/runtime/src/main/kotlin/org/jboss/resteasy/reactive/server/runtime/kotlin/FlowToPublisherHandler.kt

Reply:
Ah.  IDEA cache strikes again.  Thanks.  :)

Reply:
Yeah, the project is too big for IntelliJ to handle 😎